Mathlib/RingTheory/Ideal/Quotient/Operations.lean

```diff
@@ -429,6 +429,12 @@ theorem Quotient.liftₐ_comp (I : Ideal A) [I.IsTwoSided]
     (Ideal.Quotient.liftₐ I f hI).comp (Ideal.Quotient.mkₐ R₁ I) = f :=
   AlgHom.ext fun _ => (Ideal.Quotient.lift_mk I (f : A →+* B) hI :)
 
+theorem Quotient.span_singleton_one (I : Ideal A) [I.IsTwoSided] :
+    Submodule.span A {(1 : A ⧸ I)} = ⊤ := by
+  rw [← map_one (mk _), ← Submodule.range_mkQ I, ← Submodule.map_top, ← Ideal.span_singleton_one,
+    Ideal.span, Submodule.map_span, Set.image_singleton]
+  rfl
```

Comment:
Is it possible to replace this `rfl` by a `rw` or an `exact`?

Reply:
I don't know; this code is not written by me...

Reply:
Done.